### ISSUE TITLE
Still not quiet enough

### ISSUE THREAD

**Issue by evandrocoan** [PyCQA/pylint#2202]
I was looking for a `quiet` mode on https://github.com/PyCQA/pylint/issues/2195 (pylint: error: no such option: --quiet), but I just found out pylint is "quiet" by default. But still doing a too much output.

1. When *not everything* went fine, this new `--quiet` mode should do:
    1. **Steps to reproduce**
       Create the file `test.py`
       ```python
       class ProductionDuplicated(object):
           pass

       class ProductionDuplicated(object):
           pass
       ```

    1. **Current behavior**
       Run pylint with the command: 
       ```java
       D:\tests>pylint --disable=I,E,R,W,C,F --enable=E0102 --reports=no test.py
       ************* Module test
       test.py:6:0: E0102: class already defined line 3 (function-redefined)

       ---------------------------------------------------------------------
       Your code has been rated at -2.50/10 (previous run: 10.00/10, -12.50)


       D:\tests>
       ``` 

    1. **Expected behavior**
       Run pylint with the command: (now with `--quiet` and 1 error to show)
       ```java
       D:\tests>pylint --disable=I,E,R,W,C,F --enable=E0102 --reports=no --quiet test.py
       ************* Module test
       test.py:6:0: E0102: class already defined line 3 (function-redefined)
       D:\tests>
       ``` 

1. When *everything* went fine, this new `--quiet` mode should do:
    1. **Steps to reproduce**
       Create the file `test.py`
          ```python
          class Production(object):
              pass

          class ProductionNotDuplicated(object):
              pass
          ```

    1. **Current behavior**
       Run pylint with the command:
       ```java
       D:\tests>pylint --disable=I,E,R,W,C,F --enable=E0102 --reports=no test.py

       ---------------------------------------------------------------------
       Your code has been rated at 10.00/10 (previous run: -2.50/10, +12.50)


       D:\tests>
       ``` 

    1. **Expected behavior**

       Run pylint with the command: (now with `--quiet` and no error to show)
       ```java
       D:\tests>pylint --disable=I,E,R,W,C,F --enable=E0102 --reports=no --quiet test.py
       D:\tests>
       ``` 

### pylint --version output

```
pylint --version
pylint 2.0.0.dev1
astroid 1.6.1
Python 3.6.3 |Anaconda, Inc.| (default, Oct 15 2017, 03:27:45) [MSC v.1900 64 bit (AMD64)]
```

**Comment by PCManticore** [comment]
Yes, those were explicitly omitted from being under `--quiet/--verbose`. You can control the score with `-sn` and the whole report with `-rn`. The reason for this is that we can't be fully quiet by default, since it would mean hiding the score and the reports, and at least the score is pretty useful for newcomers to the tool. Still this means there are three options that controls how verbose pylint is, but all you really need is `-rn` for disabling everything report related. The current `--verbose` is used just for printing the configuration file that is used by pylint.
